array_reverse/array-shift.js

```diff
@@ -0,0 +1,14 @@
+const arr = [1, 2, 3, 4, 5];
+const n = 'hi';
+
+const arrayShifter = (n, arr) => {
+  let arrayShifted = arr;
+  const halfway = arr.length / 2;
+  arrayShifted.splice(halfway, 0, n);
```

Comment:
The instructions are to not use any built in methods (such as splice, pop, push, shift, unshift, slice, etc. etc.)